### PR TITLE
Added a recipe for opty.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
@conda-froge/core

This is ready to merge.

File: recipes/opty/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.8.1
```
Comment (scopatz) [inline recipes/opty/meta.yaml]: Doesn't numpy have to be `numpy. x.x` if it is a build requirement?

Comment (moorepants) [inline recipes/opty/meta.yaml]: I guess I'm confused here. If the setup.py has `install_requires` items in it, don't they have to be installed in the build? Or do they?  If not, I can remove them. Or should the build only have items that are effectively imported in the setup.py file?

Comment (moorepants) [inline recipes/opty/meta.yaml]: I removed this and it works in the latest commit.

**Comment by moorepants** [comment]
Also, ready for merging.

**Comment by scopatz** [comment]
Excellent!

**Comment by moorepants** [comment]
The merge seems to have failed on master. What do we do about that?

**Comment by scopatz** [comment]
It looks like the feedstock and package were uploaded.

**Pull request merged**